### DISCUSSION TITLE
Error when run

### DISCUSSION THREAD

**Discussion by dildardk123** [Lumieducation/Lumi#1709] [Lumi Run]
Hi, 
I download your source code, to make our desktop application using your source code.
But when i am running npm start after create build it is giving me error 
Error: Error: ENOENT: no such file or directory, open 'C:\Users\dildar\Desktop\liveWork\anchorNew\Lumi\dev-app-update.yml'
Please help me to run this project on localhost.
I have checked your package.json file it is taking "main": "build/server/main.js", Can we change main path?

**Comment by sr258** [comment]
Hi,

The missing dev-app-update.yml file should not affect startup, the issue must lie somewhere else.

It should work if you do this:

1. Clone the repo
2. Execute `npm run ci:all`
3. Execute `npm run build`
4. Execute `npm start`

You need recent recent Node and NPM version and your script execution of NPM should be done with bash and not the default Windows shell, if you are on Windows.

**Reply by dildardk123** [reply]
Hi i follow your steps, 
I am using windows system with powershell, When i am executing :- npm start then
it is showing me in cmd these lines
> lumi@0.8.0-beta.4 start
> Set DEBUG=lumi:* PORT=8080  ./node_modules/.bin/electron .

**Reply by sr258** [reply]
As I've said, you must use Bash (it comes with Git for Windows) or figure out how to replace the CLI scripts so that they work for Powershell.
